english/marshmallow.md

```diff
@@ -694,3 +694,8 @@ I consider publishing a spoiler to the new [Star Wars movie](star-wars-spoiler/s
 "What's a Heffelump?" asked Sheffield inquisitively.
 
 "I dont't know! Why should I care?" replied Donald Trump.
+
+When faced with a lot of work and impending deadlines in the (not so near) future,
+there are two options you must choose from. You may either do [this](gtd/gtd.md), or [that](procrastinate/procrastinate.md).
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.